user.py

```diff
@@ -0,0 +1,62 @@
+import re
+
+users = {}
+regex_member = "[a-zA-Z0-9- .]+$"
+regEmail = r"(^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$)"
+regPassword = r"^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$"
+
+class Users(object):
+    """
+      Users Class to handle  user functions like login and signup
+      """
+    def __init__(self, member=None, email=None, password=None):
+        """ Initializing  class instance variables"""
+        self.member = member
+        self.email = email
+        self.password = password
+
+    def member_register(self, email, member, password, cpassasword):
+        """ method that defines the elements required to create an account """
+        if re.match(regex_member, member):
```

Comment:
Your `if` statements are untidy. If a condition is met it's body is executed, else it is ignored. So you can finish one `if` statement then continue with the next